win32ss/user/user32/controls/appswitch.c

```diff
@@ -636,12 +636,12 @@ LRESULT WINAPI DoAppSwitch( WPARAM wParam, LPARAM lParam )
    }
 
    // Capture current active window.
-   SetCapture( hwndActive );
+   if (hwndActive)
+       SetCapture(hwndActive);
 
    switch (lParam)
    {
       case VK_TAB:
-         if( !CreateSwitcherWindow(User32Instance) ) goto Exit;
          if( !GetDialogFont() ) goto Exit;
```

Comment:
(Optional request): Can you take the opportunity to fix the whitespace/code-style in these lines?